Hippo.Core/Domain/Product.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace Hippo.Core.Domain
+{
+    public class Product
```

Comment:
are you going to allow products to be updated? Or just delete/create/clone? If it's just delete/create I think order linking to product is maybe worth it, even if we don't use it in the UI.  Also a create date or last updated data here would be handy

Reply:
I was going to allow update